index.d.ts

```diff
@@ -0,0 +1,25 @@
+import { FastifyPlugin } from "fastify";
+import { LevelUp } from "levelup";
+
+export interface LevelDBOptions {
+  name: string,
+  path: string
+}
+
+type LevelDBInstances {
```

Comment:
I think the syntax is wrong.
```suggestion
interface LevelDBInstances {
```